src/api/WebApi/WebApi/Controllers/WildBoarIotDataController.cs

```diff
@@ -15,9 +19,39 @@ public WildBoarIotDataController(IWildBoarIotDataService _wildBoarIotDataService
             wildBoarIotDataService = _wildBoarIotDataService;
         }
 
-        [HttpGet()]
-        public async Task<List<WildBoarIotData>> Get() =>
-            await wildBoarIotDataService.GetAsync();
+        [HttpGet("{sort?}/{format?}/{type?}/{weight?}/{occupied?}/{date_start?}/{date_end?}")]
```

Comment:
These optional parameters do not have default value, so they are not treated as optional. Please see https://code-maze.com/aspnetcore-web-api-optional-parameters/ for help.